src/components/Comment.js

```diff
@@ -57,15 +57,15 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 // return time since creation ex: a few seconds ago
-const getCreatedDuration = (createdAt) => {
+/*const getCreatedDuration = (createdAt) => {
   const seconds = moment().unix() - createdAt;
   return moment.duration(seconds, 'seconds').humanize();
-};
+};*/
 
-export const Comment = ({ comment, author, theme }) => {
+export const Comment = ({ comment, theme }) => {
   const classes = useStyles(theme);
-  const authorAge = moment(author.created_utc * 1000);
-  const commentAge = getCreatedDuration(comment.created_utc);
+  const authorAge = moment(comment.author.created_utc * 1000);
+  const commentAge = 3; // just for testing purposes
```

Comment:
was there anything broken with the getCreatedDuration function?

Reply:
Backend isn't/wasn't sending comment created time in data

Reply:
ok sounds good